README.md

```diff
@@ -180,6 +187,11 @@ This will ensure that the server executable is up-to-date before running it.
 Second we used the `--unit-testing` flag,
 which will set some testing options, create a clean new database, and load the test courses (inside the `testdata directory).
 
+Additionally, when running the server in `--unit-testing` mode,
+the server will load options from environment variables.
```

Comment:
This is misleading.
The server (or any CMD) will already load options from env variables.